lib/brisket.js

```diff
@@ -37,6 +37,8 @@ var Brisket = {
 
     $: require("./application/jquery"),
 
+    version: require("../package.json").version,
```

Comment:
Maybe we can have a version file with the version number manually updated. And we can enforce a test to check the version number is the same as the one in package.json.